syncer_server.go

```diff
@@ -24,13 +47,18 @@ func (s *PersistentSyncerServer) SetRecord(c context.Context, msg *proto.SetReco
 		}
 		return nil, err
 	}
+	s.eventsManager.notifyChange(c.Value(middleware.USER_PUBKEY_CONTEXT_KEY).(string), msg.Record)
```

Comment:
I think here the `msg.Record.Version` is the existing record version, which is then being sent to the stream. Should we emit the Record with the `newVersion`? Then the client knows the latest version